server/app/views/api/ApiDocsView.java

```diff
@@ -136,6 +136,19 @@ private DivTag contentDiv(
                 div()
                     .withClasses("items-center", "mx-6", "my-8")
                     .with(h1("API documentation"))
+                    .with(
+                        p().withClasses("my-2")
+                            .with(
+                                text(
+                                    "This page includes program-specific API documentation for"
+                                        + " Active and Draft programs. Detailed documentation about"
+                                        + " authentication, pagination, and every field's possible"
+                                        + " values is located at "),
+                                a("docs.civiform.us/it-manual/api")
+                                    .withHref("https://docs.civiform.us/it-manual/api")
+                                    .withClasses(
+                                        "text-blue-500", "underline"), // create a clickable link
```

Comment:
Nit & optional: Worth extracting these classes out into a style value that we can reference multiple times?

Reply:
Turns out this already exists! Updating to use it.